### DISCUSSION TITLE
nelua's preprocessor syntax conflicts with lua's multiline string

### DISCUSSION THREAD

**Discussion by ghost** [edubart/nelua-lang#151] [Q&A] [answered]
I want a multiline string in the nelua preprocessor code, something like
```lua
##[[

local multilinestring = [[
my
multi
string

]]
]]

```

but this won't because of ``]]``, is there any other way of doing that?

**Comment by edubart** [comment, marked as answer]
Like in Lua you can place any number of `=` between `[` `[` to make nested long strings, so you could do:

```lua
##[==[

local multilinestring = [=[
my
multi
string

]=]
]==]
```

**Reply by ghost** [reply]
Thanks
